app.py

```diff
@@ -31,23 +41,37 @@
     responses={404: {"description": "Not found"}},
 )
 
-if settings.get("BOOTSTRAP_NODE", False) is True:
-    api_v1.include_router(bootstrap_v1)
-logging.info(f"Bootstrap on this node enabled: {settings.BOOTSTRAP_NODE}")
-logging.info(f"Bootstrap ID: {settings_repository.get_fresh('BOOTSTRAP')}")
 
-if settings.get("TOKENS_NODE", True) is True and is_auth_enabled:
-    api_v1.include_router(token_v1)
-    logging.info(
-        f"Tokens on this node enabled: {settings.get('TOKENS_NODE', True)}"
-    )
+def load_endpoints():
+    # load endpoints
+    disabled_endpoints: List[str] = settings.get(
+        "DISABLE_ENDPOINTS", ""
+    ).split(":")
+    if is_auth_enabled is False:
+        disabled_endpoints.append(f"{api_v1.prefix}{token_v1.prefix}/")
+
+    for v1_endpoint in v1_endpoints:
+        endpoint_routes = v1_endpoint.routes.copy()
+        for endpoint_route in endpoint_routes:
+            route = (
+                f"{endpoint_route.methods}{api_v1.prefix}{endpoint_route.path}"
+            )
+            if route in disabled_endpoints:
+                logging.info(f"Disabled endpoint {route}")
+                v1_endpoint.routes.remove(endpoint_route)
 
-api_v1.include_router(config_v1)
-api_v1.include_router(metadata_v1)
-api_v1.include_router(targets_v1)
-api_v1.include_router(tasks_v1)
+        if f"{api_v1.prefix}{v1_endpoint.prefix}/" in disabled_endpoints:
+            logging.info(
+                f"Disabled endpoint {api_v1.prefix}{v1_endpoint.prefix}/"
+            )
+        else:
+            api_v1.include_router(v1_endpoint)
 
-rstuf_app.include_router(api_v1)
+    rstuf_app.include_router(api_v1)
+
+
+load_endpoints()
+logging.info(f"Bootstrap ID: {settings_repository.get_fresh('BOOTSTRAP')}")
```

Comment:
Are you sure there will be `bootstrap` when you import the `app.py`?

Reply:
No, but then it will log that the bootstrap is None.
```logs
repository-service-tuf-api_1     | 10:40:59 INFO Bootstrap ID: None
```